examples/graph/tool_requires/using_protobuf/myserver/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+find_package(protobuf CONFIG REQUIRED)
```

Comment:
protobuf not necessary as a direct dependency? Typically only ``myserver`` needs a ``find_package()``